corehq/apps/es/es_query.py

```diff
@@ -194,6 +194,11 @@ def add_filter(*args, **kwargs):
                 return add_filter
         raise AttributeError("There is no builtin filter named %s" % attr)
 
+    def __getitem__(self, sliced):
+        start = sliced.start or 0
+        size = sliced.stop - start
+        return self.start(start).size(size).run()
```

Comment:
Thanks. I'll look into that. 

At first that seems like it could enable bad coding where you try to loop by some index and it runs a db query through each iteration, but should probably still be supported